### ISSUE TITLE
'Extras' shown in method list summaries?

### ISSUE THREAD

**Issue by MSP-Greg** [lsegal/yard#969]
templates\default\module\html\item_summary.erb, lines 3 - 7 are
```erb
3   <% if @item.tags(:overload).size == 1 %>
4     <%= signature(@item.tag(:overload), true, !@item.attr_info) %>
5   <% else %>
6     <%= signature(@item, true, false, !@item.attr_info) %>
7   <% end %>
```
Should the #4 line change to:
```erb
4     <%= signature(@item.tag(:overload), true, false, !@item.attr_info) %>
```

Third parameter is extras, included in #6, but not in #4

Greg

**Comment by lsegal** [comment]
I would accept a PR for this.

**Comment by lsegal** [comment]
Closed by #970
